client/my-sites/plans/jetpack-plans/product-store/item-price/style.scss

```diff
@@ -58,8 +58,21 @@
 
 	.plan-price.is-original {
 		color: var(--studio-gray-50);
-		text-decoration: line-through;
 		font-weight: 600;
+		position: relative;
+		white-space: nowrap;
```

Comment:
I'm not sure the white-space attribute here is needed

Reply:
Yeah, the only problem with this method of strike-through (using pseudo-element) is that it will not work/look properly if the element wraps into two lines..  In this case the price width is so small I can't imagine it could ever wrap into two lines, but I put the `white-space: nowrap;` there just to make sure.. and to indicate that the element is not supposed to wrap.  But yes, probably not needed though..